### ISSUE TITLE
Can't transpile JS when React is installed through NPM.

### ISSUE THREAD

**Issue by treetrum** [bdkjones/CodeKit#366]
**Quick, short summary:**
If react is present inside the node_modules folder, CodeKit cannot transpile a JS file using Babel.

**Expected results:**
Transpile my JS without any issues.

**Actual results:**
JS file fails with the error:

```
Babel: Transpiling failed:

Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
ReferenceError: [BABEL] /Users/Sam/Desktop/react-babel-codekit-error/app.js: Unknown option: /Users/Sam/Desktop/react-babel-codekit-error/node_modules/react/react.js.Children. Check out http://babeljs.io/docs/usage/options/ for more information about options.

A common cause of this error is the presence of a configuration options object without the corresponding preset name. Example:

Invalid:
  `{ presets: [{option: value}] }`
Valid:
  `{ presets: [['presetName', {option: value}]] }`

For more detailed information on preset configuration, please see http://babeljs.io/docs/plugins/#pluginpresets-options. (While processing preset: "/Users/Sam/Desktop/react-babel-codekit-error/node_modules/react/react.js")
    at Logger.error (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/logger.js:41:11)
    at OptionManager.mergeOptions (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:232:20)
    at /Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:271:14
    at /Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:340:22
    at Array.map (native)
    at OptionManager.resolvePresets (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:281:20)
    at OptionManager.mergePresets (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:270:10)
    at OptionManager.mergeOptions (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:255:14)
    at OptionManager.init (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/options/option-manager.js:385:12)
    at File.initOptions (/Applications/CodeKit 3.app/Contents/Resources/engines/node_modules/babel-core/lib/transformation/file/index.js:212:65)
```

**Exact steps to reproduce:**
Try and compile the app.js file with the supplied sample project when React is present inside the node_modules folder.
If you delete React from inside node_modules, you can then transpile the JS file, reinstall React using NPM (or Yarn in my instance) and then continue until you need to re-transpile your js file.

**A link to download a simplified project or file that shows the issue:**
[react-babel-codekit-error.zip](https://github.com/bdkjones/CodeKit/files/1020705/react-babel-codekit-error.zip)

**Your configuration (any details about your system that you think might be relevant)**
![screen shot 2017-05-23 at 12 16 29 pm](https://cloud.githubusercontent.com/assets/4800689/26335851/2662234c-3fb2-11e7-99e0-fc0b360a2904.png)
![screen shot 2017-05-23 at 12 25 54 pm](https://cloud.githubusercontent.com/assets/4800689/26335983/fd7fe5c6-3fb2-11e7-867e-c3f4d31671bf.png)

**Comment by treetrum** [comment]
Possible dupe of #200 
... My bad.

**Comment by bdkjones** [comment]
This should have been resolved in 3.3. If it wasn't, please re-open.
